core/arch/arm/kernel/generic_boot.c

```diff
@@ -225,7 +237,8 @@ static void init_runtime(unsigned long pageable_part)
 	malloc_add_pool(__heap2_start, __heap2_end - __heap2_start);
 
 	hashes = malloc(hash_size);
-	IMSG("Pager is enabled. Hashes: %zu bytes", hash_size);
+	IMSG("Pager is enabled. Hashes: %zu bytes, %zukB pool",
+			hash_size, pageable_size >> 10);
```

Comment:
ok, but this is buggy. it shows the size of the pageable area, not the number of physical pages available the the pager, which was the value I was expecting.
There is already a way to get this info: from pager statistics, but it requires `CFG_WITH_STATS=y`. I think i will discard this change.